dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -195,41 +192,30 @@ private DependencyNode resolveCompileTimeDependencies(
    * @param artifacts Maven artifacts to retrieve their dependencies
    * @return dependency graph representing the tree of Maven artifacts
    */
-  public DependencyGraphResult buildLinkageCheckDependencyGraph(List<Artifact> artifacts) {
+  public DependencyGraphResult buildFullDependencyGraph(List<Artifact> artifacts) {
     ImmutableList<DependencyNode> dependencyNodes =
         artifacts.stream().map(DefaultDependencyNode::new).collect(toImmutableList());
-    return buildDependencyGraph(
-        dependencyNodes, GraphTraversalOption.FULL_DEPENDENCY_WITH_PROVIDED);
-  }
-
-  /**
-   * Finds the full compile time, transitive dependency graph including duplicates and conflicting
-   * versions.
-   */
-  public DependencyGraphResult buildCompleteGraph(Dependency dependency) {
```

Comment:
buildCompleteGraph and buildLinkageCheckDependencyGraph have been merged to buildFullDependencyGraph.